src/screens/ContactScreen/elements/Label.js

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
 import {StyleSheet, Text, View} from 'react-native';
+import {COLORS} from '../../../utils/theme';
 
 const Label = ({
   label,
-  containerBgColor = '#F2F2F2',
-  titleHeaderColor = '#F4F4F4',
-  textColor = '#4F4F4F'
+  containerBgColor = COLORS.concrete,
+  titleHeaderColor = COLORS.whiteSmoke,
+  textColor = COLORS.emperor
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for maintainability and consistency. By using color constants from a theme file, it's easier to manage and update the color scheme of the application. This also reduces the risk of typos or inconsistencies in color values.
```diff
-  containerBgColor = '#F2F2F2',
-  titleHeaderColor = '#F4F4F4',
-  textColor = '#4F4F4F'
+  containerBgColor = COLORS.concrete,
+  titleHeaderColor = COLORS.whiteSmoke,
+  textColor = COLORS.emperor
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->